content/docs/command-reference/import-url.md

```diff
@@ -200,11 +200,11 @@ produces a regular stage in `dvc.yaml`.
   from the source. The default value is `4 * cpu_count()`. Using more jobs may
   speed up the operation.
 
-- `--version-aware` - capture cloud versioning information when importing the
-  file. By default, DVC will automatically capture cloud versioning information
-  if the URL contains a cloud versioning ID. When `--version-aware` is provided
-  along with a URL that does not contain a cloud versioning ID, DVC will capture
-  the latest version of the file.
+- `--version-aware` - capture [cloud versioning] information of the current
```

Comment:
so, `--to-remote` doesn't make sense for version aware ?

Reply:
Correct